docs/usage/edsl.rst

```diff
@@ -169,7 +168,7 @@ detail:
 
   C[i, j] = \sum_{k} (A[i, k] \cdot B[k, j])
 
-We can convert this to C++ Tile code using the same correspondence as the
+We can convert this to Tile code using the same correspondence as the
 previous example: The summation sign becomes plus-assignment, the summation
 index is omitted, dimensions are given for the output tensor, and the statement
```

Comment:
statement doesn't necessarily end in a semicolon if it is written in python

Reply:
I'm sorta thinking this whole section ought to be tabbed so that there's a discussion about the differences between the mathematical representation and the form it takes in a target language.